public_html/wp-content/plugins/wc-post-types/wc-post-types.php

```diff
@@ -1300,6 +1310,36 @@ public function metabox_session_info() {
 			</select>
 		</p>
 
+		<p>
+			<label for="wcpt-session-length-hours">
+				<?php esc_html_e( 'Length:', 'wordcamporg' ); ?>
+			</label>
+
+			<select id="wcpt-session-length-hours" name="wcpt-session-length-hours" aria-label="<?php esc_html_e( 'Session Length: Hours', 'wordcamporg' ); ?>">
+				<?php for ( $i = 0; $i <= 23; $i++ ) : ?>
+					<option value="<?php echo esc_attr( $i ); ?>" <?php selected( $i, $session_length_hours ); ?>>
+						<?php echo esc_html( $i ); ?>
+					</option>
+				<?php endfor; ?>
+			</select>
+			<label for="wcpt-session-length-hours">
```

Comment:
It looks like while two labels for the same input is technically valid html, [screen readers might not read this correctly.](https://stackoverflow.com/questions/2829936/can-an-input-field-have-two-labels/3992160#3992160) A more standard approach would be a `fieldset` to wrap both inputs, and `legend` for the first label; and then you don't need the `aria-label`s either (the legend will set the context for both selects, then read their labels).

Reply:
That's a great catch, thanks! Fixed in d9bdd2acd053c97a58c4de834ef29ad117254f4f